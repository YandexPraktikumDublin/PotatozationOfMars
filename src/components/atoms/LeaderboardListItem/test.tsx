import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { LeaderboardListItem } from '.'

describe('<LeaderboardListItem />', () => {
  const position = 1
  const login = 'IvanIvanov'
  const scores = 1023

  it('should renders correct <LeaderboardListItem />', () => {
    const wrapper = shallow(
      <LeaderboardListItem position={position} login={login} scores={scores} />
    )

    expect(wrapper.contains(position)).toBeTruthy()
    expect(wrapper.contains(login)).toBeTruthy()
    expect(wrapper.contains(scores)).toBeTruthy()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
