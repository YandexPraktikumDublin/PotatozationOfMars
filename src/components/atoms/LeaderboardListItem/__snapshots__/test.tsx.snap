// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LeaderboardListItem /> should renders correct <LeaderboardListItem /> 1`] = `
<div
  className="flex justify-between border-b border-primary mb-4 dark:border-white"
>
  <span>
    1
    . 
    IvanIvanov
  </span>
  <span>
    1023
  </span>
</div>
`;
